### PR TITLE
Java - Arrays vs Lists

### PR DESCRIPTION
**Pull request by kapnobatai137**
- improved formatting

### CODE REVIEW

**Comment by enki-hq** [comment]
Rule insight:author failed for files: 
  - java/data-management/arrays/arrays-vs-lists.md

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/java/data-management/arrays/arrays-vs-lists.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/java/data-management/arrays/arrays-vs-lists.md?ref=45f941ae169fe187917ef5a03b977ab9aa579f6f)

**Comment by enki-hq** [comment]
Rule insight:footnotes failed for files: 
  - java/data-management/arrays/arrays-vs-lists.md

**Comment by kapnobatai137** [comment]
the rule might be failing because there's a `[3]` inside a code block:
```
// Fixed size array declaration, type int
int[] array = new int[3];
```

File: java/data-management/arrays/arrays-vs-lists.md [hunk 1]
```diff
@@ -29,18 +26,18 @@ links:
 
 The declaration of these structures is:
 ```java
-//Fixed size array declaration, type int
+// Fixed size array declaration, type int
 int[] array = new int[3];
```
Comment (kapnobatai137) [inline java/data-management/arrays/arrays-vs-lists.md:30]:
```suggestion
int[] array = new int[3 ];
```

**Pull request merged**